solr/solr-ref-guide/modules/indexing-guide/pages/language-analysis.adoc

```diff
@@ -692,16 +692,16 @@ On the other hand, it can reduce precision because language-specific character d
 
 == OpenNLP Integration
 
-The `lucene/analysis/opennlp` module provides OpenNLP integration via several analysis components: a tokenizer, a part-of-speech tagging filter, a phrase chunking filter, and a lemmatization filter.
+The `lucene/analysis/opennlp` package provides OpenNLP integration via several analysis components: a tokenizer, a part-of-speech tagging filter, a phrase chunking filter, and a lemmatization filter.
```

Comment:
```suggestion
The Lucene-Module `lucene/analysis/opennlp` provides OpenNLP integration via several analysis components: a tokenizer, a part-of-speech tagging filter, a phrase chunking filter, and a lemmatization filter.
```

Reply:
I looked and see _Lucene module_ used, so going to change to that...